docs-v2/integrations/integration-templates/google-mail.mdx

```diff
@@ -1,20 +1,20 @@
 ---
-title: 'Gmail API Integration Template'
-sidebarTitle: 'Gmail'
+title: 'Google Mail API Integration Template'
+sidebarTitle: 'Google Mail'
 ---
 
-## Get started with the Gmail template
+## Get started with the Google Mail template
 
 <Card title="How to use integration templates"
       href="/understand/concepts/templates"
       icon="book-open">
     Learn how to use integration templates in Nango
 </Card>
 
-<Card title="Get the Gmail template"
+<Card title="Get the Google Mail template"
       href="https://github.com/NangoHQ/nango/tree/master/integration-templates/google-mail"
       icon="github">
-    Get the latest version of the Gmail integration template from GitHub
+    Get the latest version of the Google Mail integration template from GitHub
 </Card>
```

Comment:
@bastienbeurier Should we add a button to point to the UI? It's a pity we have a way to test templates in one click and it's not advertised

Reply:
Do you envision a button to link from the docs to enable the template in the app?

Reply:
We can think about it, there is no way to point to a template without having enabled an integration but that would be nice imo